index.html

```diff
@@ -0,0 +1,76 @@
+<html>
+<body>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+
+<script src="versions.js"></script>
+
+<script>
+
+function addSeparator() {
+  $('#versionsList').append(
+    $('<li>')
+      .attr('role', 'separator')
+      .attr('class', 'divider')
+  )
+}
+
+function addVersion(version, versionName) {
+  versionName = versionName || version
+
+  $('#versionsList').append(
+    $('<li>').append(
+      $('<a>')
+        .attr('href', version + '/apidocs/index.html')
+        .attr('target', 'javadocs')
+        .text(versionName)
+    )
+  );
+}
+
+$(function() {
+
+  for (version of window.apiVersions.versions) {
```

Comment:
for.. of won't work in IE < Edge

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of#Browser_compatibility